cfme/utils/net.py

```diff
@@ -58,6 +60,39 @@ def ip_echo_socket(port=32123):
             conn.close()
 
 
+def _trying_fresh_ips(vm, delay, rounds=-1):
```

Comment:
I don't think waiting indefinitely should be the default behavior for this method.

I'm thinking a default of 3 rounds would be sufficient. Indefinite wait should be an explicit call.

Reply:
Yeah, maybe it is not a best idea to make it go forever. Perhaps it should be required parameter as the number of rounds is tightly coupled with the delay (when you change one, you likely need to change the other as well).

Reply:
@prichard77 what are your thoughts here?

Reply:
I am making it a mandatory parameter. I am making it run infinitely when one will pass `math.inf` instead of -1. `math.inf` type is `float`, but I don't think it matters.